db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player (models.Model):
+    nickname = models.CharField(max_length=255)
+    email = models.EmailField(max_length=255)
```

Comment:
Why you did not make nickname unique ?) Pay more attention to all detaild of the task 
<img width="1033" alt="image" src="https://github.com/mate-academy/py-game-models/assets/102595649/1a211b02-76db-4f84-b5af-429e032cc623">